tools/static-assets/skel-minimal/.meteor/packages

```diff
@@ -14,3 +14,4 @@ typescript              # Enable TypeScript syntax in .ts and .tsx modules
 shell-server            # Server-side component of the `meteor shell` command
 webapp                  # Serves a Meteor app over HTTP
 server-render           # Support for server-side rendering
+autoupdate-polling      # Provides a lightweight hot code push functionality
```

Comment:
Should this really be in the minimal skeleton?

Reply:
Tough one. On one hand HCP functionality is not strictly necessary to write an app, on the other hand HCP is an important part of the Meteor developer experience that up until now the minimal skeleton was missing off due to the size of the original `autoupdate` package.

I think it should be included in the minimal skeleton but I wouldn't mind leaving it off if you think it would be better.

Reply:
is it acceptable as minimal to not have updates? I don't think so. So I vote to include it.